gcc/common/config/riscv/riscv-common.c

```diff
@@ -51,25 +51,27 @@ struct riscv_implied_info_t
 {
   const char *ext;
   const char *implied_ext;
+  unsigned xlen_requirement;
 };
 
 /* Implied ISA info, must end with NULL sentinel.  */
 riscv_implied_info_t riscv_implied_info[] =
 {
-  {"d", "f"},
-  {"k", "zkn"},
-  {"k", "zkr"},
-  {"zkn", "zkne"},
-  {"zkn", "zknd"},
-  {"zkn", "zknh"},
-  {"zkn", "zkg"},
-  {"zkn", "zkb"},
-  {"zks", "zksed"},
-  {"zks", "zksh"},
-  {"zks", "zkg"},
-  {"zks", "zkb"},
-  {"p", "zpn"},
-  {"p", "zpsf"},
+  {"d", "f", 0},
+  {"k", "zkn", 0},
+  {"k", "zkr", 0},
+  {"zkn", "zkne", 0},
+  {"zkn", "zknd", 0},
+  {"zkn", "zknh", 0},
+  {"zkn", "zkg", 0},
+  {"zkn", "zkb", 0},
+  {"zks", "zksed", 0},
+  {"zks", "zksh", 0},
+  {"zks", "zkg", 0},
+  {"zks", "zkb", 0},
+  {"p", "zpn", 0},
+  {"p", "zprv", 64},
+  {"p", "zpsf", 0},
   {NULL, NULL}
```

Comment:
If this logic stays, I expect `{NULL, NULL, NULL}` or `{NULL, NULL, 0}` here.
It expands to zeros anyway by the power of C, yet it seems to be 'right' to explicitly place this zero.